nncore/models/loss/CEwithstat.py

```diff
@@ -4,11 +4,11 @@
 import torch
 
 
-class CE(nn.Module):
-    """Some Information about CE"""
+class CEwithstat(nn.Module):
+    r"""CEwithstat is warper of cross-entropy loss"""
```

Comment:
I wonder if it is necessary to add `r` before the description (because I don't see any `r` in other descriptions).

Reply:
Well, I think it does not relate to PR so I think it ok.
Just note this in the future for unifying.

Reply:
I think finalize documentation in other PR is better idea, thanks for your opinion